src/raiden_libs/cli.py

```diff
@@ -100,7 +98,11 @@ def call_with_opened_keystore(**params: Any) -> Callable:
             params["private_key"] = _open_keystore(
                 params.pop("keystore_file"), params.pop("password")
             )
-            return func(**params)
+            try:
```

Comment:
Should this go into `call_with_opened_keystore` or a new function?

Reply:
I think this is a good place for it. The name `call_with_opened_keystore` could use some updating, though. Something like `call_with_common_options_initialized`. But that's not a great name, either.

Reply:
Not perfect, but better. And I have nothing better to propose...